diff --git a/node_modules/@types/parse/index.d.ts b/node_modules/@types/parse/index.d.ts
index 26a3a41..d1f7e45 100755
--- a/node_modules/@types/parse/index.d.ts
+++ b/node_modules/@types/parse/index.d.ts
@@ -179,7 +179,7 @@ declare global {
         interface Pointer {
             __type: string;
             className: string;
-            objectId: string;
+            id: string;
         }
 
         interface AuthData {
